packages/bem-twig-extension/lib/bem-twig-extension.js

```diff
@@ -5,12 +5,19 @@ module.exports = bemTwigExtension;
 function bemTwigExtension(Twig) {
   Twig.extendFunction("bem", function(base_class, modifiers = [], blockname = '', extra = [], attributes = '') {
     let classes = [];
-    
+
+    // Helper function to sanitize and validate class names
+    function sanitizeClassName(className) {
+      // Strips out initial numbers. Replace non-matches with spaces, does strip non-english characters
+      const sanitized = className.match(/-?[_a-zA-Z]+[_a-zA-Z0-9-]*/g);
```

Comment:
@codechefmarc I think the approach for this is to take it to a Engineering Team meeting and get some other eyes on this before merging.  I remember we had 3 options for the regex here, I don't remember if this is the one we landed on or not but seeing as it is a breaking change I would make sure we bump the major version and advertise that this version could break class names and thus the look and feel of a site for sites that used characters that are no longer supported.